test/test_podman_baseline.sh

```diff
@@ -78,7 +78,71 @@ echo $image
 ########
 # Run container and display contents in /etc
 ########
-podman run $image ls -alF /etc
+podman run --rm $image ls -alF /etc
+
+########
+# Test networking, bind mounting a file, stdin/stdout redirect
+########
+txt1="Hello, Podman"
+echo "$txt" > /tmp/hello.txt
+podman run -d --name myweb -p 8080:80 -w /var/www -v /tmp/hello.txt:/var/www/index.txt busybox httpd -f -p 80
+echo $txt | podman exec -i myweb sh -c "cat > /var/www/index2.txt"
+txt2=$( podman exec myweb cat /var/www/index2.txt )
+[ "x$txt1"=="x$txt2" ] && echo "PASS"
+txt2=$( podman run --rm --net host busybox wget -qO - http://localhost:8080/index.txt )
+[ "x$txt1"=="x$txt2" ] && echo "PASS"
+txt2=$( podman run --rm --net host busybox wget -qO - http://localhost:8080/index2.txt )
```

Comment:
needs `run -d`

Reply:
no it does not. I'm not taking the container id, I'm testing the output of `wget -qO -` , passing -d would make it go background and no output is stored in `txt2`

Reply:
When I ran it, the test hung there for me, that's why I thought it needed the `-d`...

Reply:
please try now